src/main/java/org/broadinstitute/hellbender/utils/SimpleInterval.java

```diff
@@ -36,6 +36,60 @@ public SimpleInterval(final String contig, final int start, final int end){
         this.end = end;
     }
 
+    /**
+     * Makes an interval by parting the string. The required format is
+     * contig or contig:start or contig:start-end
+     *
+     * example:
+     *
+     * 'chr2', 'chr2:1000000' or 'chr2:1,000,000-2,000,000'
+     */
+    public SimpleInterval(final String string){
+        Locatable loc = parse(string);
+        this.contig = loc.getContig();
+        this.start = loc.getStart();
+        this.end = loc.getEnd();
+    }
+
+    private static Locatable parse(final String str){
+        // 'chr2', 'chr2:1000000' or 'chr2:1,000,000-2,000,000'
+        //System.out.printf("Parsing location '%s'%n", str);
+
+        final String contig;
+        final int start;
+        final int stop;
+
+        final int colonIndex = str.lastIndexOf(":");
```

Comment:
Declare special delimiters `':'` and `'-'` as named constants at the top of the class.